CHANGELOG.md

```diff
@@ -8,7 +8,13 @@ All notable changes to this project will be documented in this file.
 
 - Support row filters and column masks in Rego rules ([#559]).
 
+### Changed
+
+- BREAKING: Change the username that triggers graceful shutdown from `admin` to `graceful-shutdown-user` for more expressiveness (e.g. in the Trino policies).
+  This is breaking as users needs to make sure the user `graceful-shutdown-user` has the needed permissions to initiate a graceful shutdown ([#573]).
```

Comment:
```suggestion
  This is a breaking change because users need to ensure that the user `graceful-shutdown-user` has the required permissions to initiate a graceful shutdown ([#573]).
```